Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM python:3
 
 COPY keycloak-metrics-spi.jar ./
+COPY openshift4-extension-6.0.1.jar ./
```

Comment:
I'm not sure what the best practice here is but the idea was to strip any version information from the binaries and bundle one defined version of a plugin with a specific tag of the init container.